OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -44,70 +54,48 @@ public class CloakInfo : UpgradableTraitInfo
 		[Desc("The upgrades to grant to self while cloaked.")]
 		public readonly string[] WhileCloakedUpgrades = { };
 
-		public override object Create(ActorInitializer init) { return new Cloak(init.Self, this); }
+		public override object Create(ActorInitializer init) { return new Cloak(this); }
 	}
 
-	public class Cloak : UpgradableTrait<CloakInfo>, IRenderModifier, INotifyDamageStateChanged, INotifyAttack, ITick, IVisibilityModifier, IRadarColorModifier, INotifyCreated
+	public class Cloak : UpgradableTrait<CloakInfo>, IRenderModifier, INotifyDamageStateChanged,
+		INotifyAttack, ITick, IVisibilityModifier, IRadarColorModifier, INotifyCreated
```

Comment:
`ISync` is missing here. Not really your fault. Thought we had a lint test for it.